docs/docsite/rst/user_guide/playbooks_conditionals.rst

```diff
@@ -173,15 +173,17 @@ Or with a role::
            when: ansible_os_family == 'Debian'
 
 You will note a lot of 'skipped' output by default in Ansible when using this approach on systems that don't match the criteria.
-Read up on the 'group_by' module in the :doc:`modules` docs for a more streamlined way to accomplish the same thing.
+In many cases the ``group_by`` module (see :doc:`modules`) can be a more streamlined way to accomplish the same thing; see
```

Comment:
We are moving away from `:doc:` links, please use a `:ref:` link here - I suggested `:ref:'group_by_module'` so the user can jump straight to the module documentation for that specific module.

Reply:
Done